docs/advanced-users.md

```diff
@@ -0,0 +1,10 @@
+# Advanced Users
+
+Note that using Fleet outside of Rancher is highly discouraged for any users who do not need to perform advanced actions. However, there are some advanced use cases that may need to be performed outside of Rancher, and this section will highlight these cases. 
```

Comment:
Perhaps, we should add a note saying something along the lines of the following: "many advanced use cases are covered under what's known as Standalone Fleet, or Fleet without Rancher"

Reply:
That's a good point, I will leave @jtravee to address

Reply:
Reworded, thanks!